api/src/main/java/org/openmrs/util/OpenmrsConstants.java

```diff
@@ -854,6 +860,8 @@ public static String getOpenmrsProperty(String property) {
 		props.add(new GlobalProperty(GP_PASSWORD_MINIMUM_LENGTH, "8",
 		        "Configure the minimum length required of all passwords"));
 		
+		props.add(new GlobalProperty(GP_PASSWORD_RESET_VALIDTIME, "600000", "Default of 10 minutes for which the password reset is valid"));
```

Comment:
The description should be better, it needs to say what the global property is for and not its default value, something like **Specifies the duration of time in seconds for which a password reset token is valid, the default value is 10 minutes and the allowed values range from 1 to 12hrs**